### DISCUSSION TITLE
Cannot access any data from Ergast

### DISCUSSION THREAD

**Discussion by tmalik1116** [theOehrly/Fast-F1#632] [Q&A] [answered]
Hi everyone,

I have some experience with FastF1 already from a project I am working on. I know that FastF1 needs to fall back on Ergast to fetch data older than 2018, but I can't seem to get it to work for me and I couldn't figure out what I was doing wrong from the documentation.

This is the code I am using to try and get a session:
`session = fastf1.get_session(year=2017, 
                             gp='australia', 
                             identifier="Q",
                             backend='ergast')
session.load()`

And this is the output I get:
`req         WARNING     DEFAULT CACHE ENABLED! (6.96 GB) C:\Users\tmalik\AppData\Local\Temp\fastf1
core           INFO     Loading data for Australian Grand Prix - Qualifying [v3.4.0]
req            INFO     No cached data found for session_info. Loading data...
_api           INFO     Fetching session info data...
logger      WARNING     Failed to load session info data!
core        WARNING     Cannot load laps, telemetry, weather, and message data because the relevant API is not supported for this session.
core           INFO     Finished loading data for 20 drivers: ['44', '5', '77', '7', '33', '8', '19', '55', '26', '3', '11', '27', '14', '31', '9', '36', '20', '2', '18', '30']`

I basically need to repeat what I did for 2018-2024 and get each qualifying session of the year in order to add to a dataset. Any help would be appreciated!

**Comment by theOehrly** [comment, marked as answer]
The only data you will get is ``Session.results``. There is currently no support for loading the whole laps data from Ergast. The error message that you see is to be expected for sessions before 2018. Does this answer your question already?

**Reply by tmalik1116** [reply]
Thanks, that's all I needed to know. I was wondering if the data was accessed in the same way as for the f1timing data.
